static/src/stylesheets/module/content/_article-immersive.scss

```diff
@@ -22,17 +22,32 @@
     width: 100%;
     // Remove inline spacing
     display: block;
+    // This is for the fallback;
+    min-height: 400px;
```

Comment:
Great catch! It is on two different elements, but I only needed to put it on this one.